src/app/panels/terminal.js

```diff
@@ -201,9 +201,7 @@ class Terminal extends Plugin {
     setInterval(async () => {
       try {
         self._view.pendingTxCount.innerHTML = await self.call('udapp', 'pendingTransactionsCount')
-      } catch (err) {
-        console.log('Terminal try to call "udapp".')
-      }
+      } catch (err) {}
```

Comment:
Maybe we should add a TODO + issue to remove this try and create an event instead

Reply:
issue https://github.com/ethereum/remix-ide/issues/2191